packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch

```diff
@@ -25,7 +25,7 @@
    name: Kiali
    url: https://kiali.io
 -name: kiali-server
-+name: rancher-kiali-server
```

Comment:
why did we change the name?

Reply:
Seems like a mistake while updating. Will revert the name to the `rancher-kiali-server`.